modules/bmc/ipmiscanner.rb

```diff
@@ -0,0 +1,82 @@
+require 'bmc/basescanner'
+require 'concurrent'
+require 'bmc/bmc_plugin'
+
+module Proxy
+  module BMC
+    class IPMIScanner < BaseScanner
+      include Proxy::Log
+      include Proxy::Util
+
+      # This is an IPMI ping, not an ICMP ping.
+      def address_pings?(address)
+        begin
+          socket = UDPSocket.new
+        rescue Errno::EMFILE
+          logger.warn "IPMIScanner: Ran out of free file descriptors while creating UDPSocket! Consider increasing file open limit."
+          retry
+        end
+        socket.connect(address.to_s, 623)
+        socket.send([0x6, 0x0, 0xff, 0x6, 0x0, 0x0, 0x11, 0xbe, 0x80, 0x0, 0x0, 0x0].pack('C*'), 0)
```

Comment:
Could you add a comment describing the contents of this packet, together with a link to some docs about the call this is making?